dChatFilter/CMakeLists.txt

```diff
@@ -0,0 +1,4 @@
+set(DCHATFILTER_SOURCES "dChatFilter.cpp")
+
+add_library(dChatFilter ${DCHATFILTER_SOURCES})
```

Comment:
We should probably explicitly specify `STATIC` or `SHARED` here if we don't want this to be influenced by the value `BUILD_SHARED_LIBS`. Traditionally DLU has been built with almost entirely static linking, but we could change this as there are several shared dependencies between the server executables.

Reply:
Agreed.  Let's not leave anything up to randomness and this is a good chance to reduce the binary size a touch for the libs used across all the binaries like raknet, dCommon, dNet etc.

Reply:
Yeah that is a good idea. Will do.